css/Category.css

```diff
@@ -0,0 +1,50 @@
+.Category .box-container {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(30rem, 1fr));
+	gap: 1.5rem;
+
+}
+
+.Category .box-container .box {
+	background: var(--bg-color-1);
+	border-radius: .5rem;
+	box-shadow: var(--box-shadow);
+	padding: 2rem;
+	position: relative;
+}
+
+.Category .box-container .box .image {
```

Comment:
don't use capitalized classes in html/css.

Reply:
Thank you for bringing my attention I will fix it for sure!